ui/app/serializers/pki/issuer.js

```diff
@@ -1,11 +1,28 @@
+import { parseCertificate } from 'vault/helpers/parse-pki-cert';
 import ApplicationSerializer from '../application';
 
 export default class PkiIssuerSerializer extends ApplicationSerializer {
-  // rehydrate each issuer model so all model attributes are accessible from the LIST response
+  primaryKey = 'issuer_id';
+
+  normalizeResponse(store, primaryModelClass, payload, id, requestType) {
+    if (payload.data.certificate) {
+      // Parse certificate back from the API and add to payload
+      const parsedCert = parseCertificate(payload.data.certificate);
+      const data = { issuer_ref: payload.issuer_id, ...payload.data, ...parsedCert };
+      const json = super.normalizeResponse(store, primaryModelClass, { ...payload, data }, id, requestType);
+      return json;
+    }
+    return super.normalizeResponse(...arguments);
+  }
+
+  // rehydrate each issuers model so all model attributes are accessible from the LIST response
   normalizeItems(payload) {
     if (payload.data) {
       if (payload.data?.keys && Array.isArray(payload.data.keys)) {
-        return payload.data.keys.map((key) => ({ id: key, ...payload.data.key_info[key] }));
+        return payload.data.keys.map((issuerRef) => ({
+          issuer_id: issuerRef,
```

Comment:
oh nvm - I see you use `issuer_id` here - do we want a consistent id key name? 🤔

Reply:
I was trying to keep consistent with the docs, which refer to `issuer_ref` as the path variable (and the keys that come on the list endpoint) but returns `issuer_id` on the payload 🤷 I'm fine either way, especially if the way I have it is confusing

Reply:
Oh I see! That makes sense. I think the docs refer to the path variable as `:key_ref` as well which i just ignored in favor of the parameter on the response 😵‍💫 

I think I'm preferable to matching payload parameters. For [keys](https://developer.hashicorp.com/vault/api-docs/secret/pki#update-key) I understanding having `key_ref` as a path placeholder, but then I think the parameter below should be `key_id`, or whatever the endpoint expects/returns